aten/src/ATen/native/cuda/UnaryComplexKernels.cu

```diff
@@ -21,15 +21,41 @@ __host__ __device__ static inline scalar_t angle_wrapper(scalar_t v) {
 
 template<typename T>
 __host__ __device__ static inline c10::complex<T> angle_wrapper(c10::complex<T> v) {
-  return std::arg(v);
+  return c10::complex<T>({ std::arg(v) });
```

Comment:
Is it required to be like this? Seems like an odd way of constructing non-sequence object. Does 
`c10::complex<T>(std::arg(v));` or `c10::complex<T>{std::arg(v)};`
work?

Reply:
It works. I've updated it! :)